test/algorithms/Dichotomy.jl

```diff
@@ -261,7 +261,7 @@ function test_time_limit()
     )
     MOI.optimize!(model)
     @test MOI.get(model, MOI.TerminationStatus()) == MOI.TIME_LIMIT
-    @test MOI.get(model, MOI.ResultCount()) > 0
+    @test MOI.get(model, MOI.ResultCount()) == 0
```

Comment:
I think this behavior was because we checked for the time limit only after solving the ideal point and the first point, so there was always a solution. No problem changing it.